.github/workflows/python-package.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
Let's keep this at ubuntu-latest for now. We'll add windows/Mac later

Reply:
It looks like the error is related to running mac jobs: https://github.com/actions/virtual-environments/issues/841 so changing this back to previous should help.

Reply:
runs-on cannot take a list of words as input. a list can only be given to a matrix keyword. I have changed this file in climt to show how it should actually be done. but even then for now we should only do ubuntu.

Reply:
ok